### PR TITLE
feat: achievements section

### PR DESCRIPTION
**Pull request by nata-kab**
## Jira issue

JIRA issue: [FSU-145](https://klau.atlassian.net/browse/FSU-145)

## Describe your changes

This PR introduces the Achievements section.

## Checklist before requesting a review

- [x] Checked which branch you would merge.
- [x] Checked if the code follows the style guidelines of this project (use eslint and prettier).
- [x] Performed a self-review of code (bugs, clean code rules, etc).
- [x] Checked if added variables have unique names.
- [x] Checked if added code/function is not a duplicate. 
- [x] Tests have been added if possible.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: packages/frontend/app/templates/HomePage/components/Achievements/Achievements.jsx [hunk 1]
```diff
@@ -9,17 +11,37 @@ const Achievements = ({
...
-      🏡 {animalsNumber} Znalezionych domów
-    </Typography>
-    <Typography variant="h4" as="h3">
-      🏡 {adoptedAnimalsNumber + animalsNumber} Uratowanych zwierząt
-    </Typography>
-  </Styled.TextContainer>
+  <Styled.Achievements>
+    <TextBanner
+      heading="Nasze osiągnięcia"
+      subtitle="FUNDACJA W LICZBACH"
```
Comment (im-klau) [inline packages/frontend/app/templates/HomePage/components/Achievements/Achievements.jsx:17]:
```suggestion
      subtitle="Fundacja w liczbach"
```

File: packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js [hunk 2]
```diff
@@ -1,5 +1,51 @@
...
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    gap: 80px;
+    padding: 100px 56px;
+  }
+  ${({ theme }) => theme.breakpoints.mobileLg} {
+    gap: 60px;
+    padding: 60px 24px;
+  }
+  ${({ theme }) => theme.breakpoints.mobileSm} {
+    padding: 60px 14px;
```
Comment (im-klau) [inline packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js:49]: You don't need to set tablet and mobile horizontal paddings as the `Container` that's around this section adds them anyway. So now, you have duplicated paddings.

**Review by im-klau — CHANGES_REQUESTED** [review]
I've found just one, small issue but other than that the section looks perfect ❤️ Amazing work! 🙌

File: packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js [hunk 3]
```diff
@@ -1,5 +1,51 @@
...
+  ${({ theme }) => theme.breakpoints.tabletLg} {
+    grid-auto-flow: row;
+    gap: 80px;
+  }
+  ${({ theme }) => theme.breakpoints.mobileLg} {
+    gap: 60px;
+  }
+`
+export const Description = styled.div`
+  display: flex-inline;
```
Comment (AmandaKr) [inline packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js:20]:
```suggestion
  display: inline-flex;
```

File: packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js (hunk 2 above)
Comment (nata-kab) [reply packages/frontend/app/templates/HomePage/components/Achievements/Achievements.styled.js]: Done 😀

**Review by paulgrym — APPROVED** [review]
🚀

**Review by im-klau — APPROVED** [review]

**Pull request merged**